.idea/geneticApp.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file like the following files should not be in the repository. You should put `.idea` to `.gitignore` to ignore such files.

Reply:
Also dont include the binaries here like the above one.